### ISSUE TITLE
[v469b/c] Replace `ucc dumpint` and `ucc compareint` with Unrealv227j's `ucc dumpint`

### ISSUE THREAD

**Issue by NeonKnightOA** [OldUnreal/UnrealTournamentPatches#843]
Take the following lines (extracted from `UnrealShare.est`):
```
[UnrealTeamScoreBoard]
; EN: TeamName[0]="Red Team: "
TeamName[0]="Equipo Rojo: "
; EN: TeamName[1]="Blue Team: "
TeamName[1]="Equipo Azul: "
; EN: TeamName[2]="Green Team: "
TeamName[2]="Equipo Verde: "
; EN: TeamName[3]="Gold Team: "
TeamName[3]="Equipo Dorado: "
```

Currently, when `ucc dumpint UnrealShare.u` is ran, it outputs the following (taken from UnrealShare.int): 
```
[UnrealTeamScoreBoard]
TeamName[0]=Red Team: 
TeamName[1]=Blue Team: 
TeamName[2]=Green Team: 
TeamName[3]=Gold Team: 
```

When this is outputted to an already localized file (suppose, `UnrealShare.est`) by using `ucc compareint`, it outputs the localized lines in the appropriate format:
```
[UnrealTeamScoreBoard]
TeamName[0]=Equipo Rojo: 
TeamName[1]=Equipo Azul: 
TeamName[2]=Equipo Verde: 
TeamName[3]=Equipo Dorado: 
```

A nice addition of `ucc dumpint` on v227j is that, **in all languages not named English, untranslated lines are marked with `[ENG]`**. This has proven to be a very helpful tool to locate untranslated strings.
```
TeamName[3]=[ENG]Gold Team: 
```

In several cases (especially files with starting or ending spaces -I'm looking at you `Editor.*`-) **the double quotes are needed**. Otherwise, the files end up looking as a mess (looking at you, `[UMenuCustomizeClientWindow]` section of `UMenu.*` and how much you desorganize the Controls menu).

Additionally, **for all languages not named English, the original English lines are needed** so localizers know if the lines require maintenance or not. Something like this (taken from Buggie's computer-generated templates):
```
[UnrealTeamScoreBoard]
; EN: TeamName[0]="Red Team: "
TeamName[0]="Equipo Rojo: "
(...)
; EN: TeamName[3]="Gold Team: "
TeamName[3]=[ENG]"Gold Team: "
```
Long story short, is there any chance for `ucc dumpint` to produce something like the above, like in Unreal v227j?

**Comment by SeriousBuggie** [comment]
Technically it is quite possible, but locate not in SDK, So only core devs help you here.

**Comment by NeonKnightOA** [comment]
I don't have access to the SDK repository. Not sure what I could contribute with there.

**Comment by NeonKnightOA** [comment]
This is already fixed. Thank you!
